src/utils/oauth/GithubProvider.ts

```diff
@@ -123,7 +123,7 @@ const GithubProvider: IProvider<boolean> = {
         /* GET request to get the user used for OAuth */
         await fetch(`${providerConfig.urls.usernameUrl}`, {
           headers: {
-            "authorization": "Token " + accessToken,
+            authorization: "Token " + accessToken,
```

Comment:
Why has this change been reverted?
```suggestion
            "authorization": "Token " + accessToken,
```

Reply:
In case we have this problem in the future again, add "// prettier-ignore" before the code part, to disable prettier.